tools/cloud-workstations/Dockerfile

```diff
@@ -40,3 +40,16 @@ RUN pip install --no-cache-dir --upgrade pip && \
     pip install --no-cache-dir -r https://raw.githubusercontent.com/SchedMD/slurm-gcp/master/scripts/requirements.txt && \
     pip install --no-cache-dir -r https://raw.githubusercontent.com/GoogleCloudPlatform/hpc-toolkit/main/tools/cloud-build/requirements.txt && \
     rm -rf ~/.cache/pip/*
+
+ARG HPC_TOOL_URL=https://raw.githubusercontent.com/GoogleCloudPlatform/hpc-toolkit/main/tools/cloud-workstations
+ARG HPC_FILE=configure-hpc-toolkit.sh
+ARG HPC_CONF_FILE=030_configure-hpc-toolkit.sh
+
+# Get the HPC config files and store them in the correct locations for startup
+RUN wget $HPC_TOOL_URL/$HPC_FILE && \
```

Comment:
@cboneti I'm not sure which URL/branch to use here.  In the example above, the main branch is used, but until this is merged from develop into main, it will be incorrect.